executable-spec/src/Cardano/Ledger/Spec/STS/Update/Data.hs

```diff
@@ -65,9 +65,19 @@ data Confidence = For | Against | Abstain
   deriving (Eq, Ord, Show, Generic, HasTypeReps)
 
 -- | Records the voting result for a specific software update (SIP/UP)
-data VotingResult
+data VotingResult =
+  VotingResult { stakeInFavor :: !Word64
+               , stakeAgainst :: !Word64
+               , stakeAbstain :: !Word64
+               }
   deriving (Eq, Ord, Show)
 
+-- | Voting threshold
+type VThreshold = Word
```

Comment:
Shouldn't we define `newtype` wrappers for `VThreshold` and `Stake`?

Reply:
Both of them are simple unsigned integers. That is why I thought that an alias would be enough.
I though we use `newtype`, when we want to make the new type an instance of a type class.

`VThreshold` is a `Word` because it takes values from `[0..100]`